packages/wrangler/src/pages/publish.tsx

```diff
@@ -58,12 +58,12 @@ export function Options(yargs: CommonYargsArgv) {
 			},
 			bundle: {
 				type: "boolean",
-				default: false,
+				default: undefined,
```

Comment:
why are we setting these defaults to `undefined`. My understanding was that crr impl keeps things in line with wrangler proper. Am I missing some context for these changes? If so, perhaps a short/sweed description of this PR could help :)

Reply:
yeah `undefined` is not the best 😕 

it is here as a hack so that we can check if the values are provided or not (because yargs just gives us the default and we have no (non-hacky) way to check if that's coming from the user or not), it is so that we can do the following:
```
	if (bundle !== undefined && noBundle !== undefined) {
		if (bundle === noBundle) {
```

Reply:
by the way, @petebacondarwin suggested not to check if we have conflicting options and just let yargs do its thing (which is taking as valid only the last one)

if we go with that we can revert to default booleans here and remove the check (and the test for it) :slightly_smiling_face: 

@GregBrimble how does that sound to you?

Reply:
> by the way, @petebacondarwin suggested not to check if we have conflicting options and just let yargs do its thing (which is taking as valid only the last one)
> 
> if we go with that we can revert to default booleans here and remove the check (and the test for it) 🙂
> 
> @GregBrimble how does that sound to you?

I am also in agreement with Petes take on this, especially hopefully avoiding `undefined` hackery lol

Reply:
yeah... I have mixed feeling about that... but also the `undefined` hack (or another hack I came up with as well) aren't much better, so I think that I don't really have a strong opinion on this one, it's all good for me :slightly_smiling_face: